docs/src/main/sphinx/installation/deployment.md

```diff
@@ -35,16 +35,16 @@
 
 ### Java runtime environment
 
-Trino requires a 64-bit version of Java 17, with a minimum required version of 17.0.3.
-Earlier major versions such as Java 8 or Java 11 do not work.
-Newer major versions such as Java 18 or 19, are not supported -- they may work, but are not tested.
+Trino requires a 64-bit version of Java 21, with a minimum required version of 21.0.1.
+Earlier major versions such as Java 8, Java 11 or Java 17 do not work.
+Newer major versions such as Java 22, are not supported -- they may work, but are not tested.
 
 We recommend using the Eclipse Temurin OpenJDK distribution from
 [Adoptium](https://adoptium.net/) as the JDK for Trino, as Trino is tested
 against that distribution. Eclipse Temurin is also the JDK used by the [Trino
 Docker image](https://hub.docker.com/r/trinodb/trino).
 
-If you are using Java 17 or 18, the JVM must be configured to use UTF-8 as the default charset by
+If you are using old Trino release with Java 17 or 18, the JVM must be configured to use UTF-8 as the default charset by
```

Comment:
This whole paragraph should now be removed since none of these versions are supported. And users of old releases .. should read the old docs.

Reply:
ok, I'll drop it